docs/src/main/paradox/release-notes/1.0.md

```diff
@@ -1,8 +1,31 @@
 # Alpakka Kafka 1.0
```

Comment:
Should we call the page and the title `1.0.x` like it is is akka-http?

We could add a redirect from the current `1.0.md` to the new `1.0.x.md`.